yaml/testlist.yaml

```diff
@@ -76,23 +85,27 @@
     +directed_instr_4=riscv_cache_line_stress_instr_stream,4
     +directed_instr_5=riscv_multi_page_load_store_instr_stream,4
   rtl_test: core_base_test
+  no_post_compare: 0
 
 - test: riscv_rand_jump_test
   description: >
     Jump among large number of sub-programs, stress testing iTLB operations.
   iterations: 2
+  no_iss: 0
   gen_test: riscv_instr_base_test
   gen_opts: >
     +instr_cnt=15000
     +num_of_sub_program=20
     +directed_instr_0=riscv_load_store_rand_instr_stream,8
   rtl_test: core_base_test
+  no_post_compare: 0
 
 - test: riscv_mmu_stress_test
   description: >
     Test with different patterns of load/store instructions, stress test MMU
     operations.
   iterations: 2
+  no_iss: 0
```

Comment:
can you make no_iss and no_compare optional? For most tests, this doesn't need to be specified. Maybe just need to modify the parsing script a little bit.

Reply:
sure, will do that